index.html

```diff
@@ -129,7 +129,61 @@ <h2 class="ui divider horizontal">Ils l'utilisent déjà</h2>
         'contract'
       ],
       searchFullText: true,
-      source: data
+      source: data,
+      templates: {
+        message: function(type, message) {
+          html = '<p class="searchError">Aucune API ne correspond à votre recherche</p>'
+          + '<a href="' + action.url + '" class="action">'
+          +   action.text
+          + '</a>';
+          return html;
+        },
+        standard: function(response, fields) {
+          var
+            html = ''
+          ;
+          if(response[fields.results] !== undefined) {
+
+            // each result
+            $.each(response[fields.results], function(index, result) {
+              if(result[fields.url]) {
+                html  += '<a class="result" href="' + result[fields.url] + '">';
+              }
+              else {
+                html  += '<a class="result">';
+              }
+              if(result[fields.image] !== undefined) {
+                html += ''
+                  + '<div class="image">'
+                  + ' <img src="' + result[fields.image] + '">'
+                  + '</div>'
+                ;
+              }
+              html += '<div class="content">';
+              if(result[fields.price] !== undefined) {
+                html += '<div class="price">' + result[fields.price] + '</div>';
+              }
+              if(result[fields.title] !== undefined) {
+                html += '<div class="title">' + result[fields.title] + '</div>';
+              }
+              if(result[fields.description] !== undefined) {
+                html += '<div class="description">' + result[fields.description] + '</div>';
+              }
+              html  += ''
+                + '</div>'
+              ;
+              html += '</a>';
+            });
+            if(action) {
```

Comment:
@MattiSG : I actually took the template from [semanticUI](https://github.com/Semantic-Org/Semantic-UI/blob/42e41c2699691ff59c148a41a326a9b874a0f08e/dist/components/search.js#L1295) and I changed a the action part.